abacus-integration-test/e2e/Attributes.spec.ts

```diff
@@ -20,11 +20,11 @@ test.describe('<Attributes/>', () => {
 
   test.beforeEach(async ({ page, playwright, authority }) => {
     await authorize(page);
-    authToken = await getAccessToken(page)
-    await page.goto('/attributes');
-    // click the token message to close it and overcome potential overlapping problem
-    const notificationElement = await page.locator(selectors.tokenMessage);
-    await notificationElement.click();
+    authToken = await getAccessToken(page);
+
+    await page.getByRole('link', { name: 'Attributes' }).click();
+    await page.waitForURL('http://localhost:65432/attributes');
```

Comment:
Is it possible to avoid setting full url in such a lot of places (to avoid multiple fixes in case of possible change)? 
Maybe this method work with baseUrl (mean setting like '/attributes')? Or maybe use regex ('**/attributes') or even const for base part?